### PR TITLE
Yurii shevchuk

### PR DESCRIPTION
**Pull request by jurec72**
# Homework - find "files"
* did two ways

### CODE REVIEW

**Comment by kykaparov** [comment]
Yurii, could you create a branch like yurii/hw1, yurii/hw2, yurii/hw3?
You are collecting all your changes in one branch, and it shows previous homework too

File: 04-JavaInputOutput/src/main/java/softClass/homework/yurii_shevchuk/ShoppingCart.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
+package softClass.homework.yurii_shevchuk;
+
+import java.util.Arrays;
+import java.util.stream.Stream;
+
+public class ShoppingCart {
```
Comment (kykaparov) [inline 04-JavaInputOutput/src/main/java/softClass/homework/yurii_shevchuk/ShoppingCart.java:6]: Thanks, Yurii for your activities and homework. 👍 
Could you push this homework to WednesdayHomeworks repo?

File: 04-JavaInputOutput/src/main/java/homework/yurii_shevchuk/Reader.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Reader {
+    public static void main(String[] args) throws IOException {
+//write a code which finds how many words "files" are in Readme.md of module 04 -JavaInputOutput
+        //1
+        String lines = Files.lines(Paths.get("04-JavaInputOutput/README.md")).collect(Collectors.joining());
```
Comment (kykaparov) [inline 04-JavaInputOutput/src/main/java/homework/yurii_shevchuk/Reader.java:21]: Great 👍 solution

File: 04-JavaInputOutput/src/main/java/softClass/homework/yurii_shevchuk/ShoppingCart.java (hunk 1 above)
Comment (jurec72) [reply 04-JavaInputOutput/src/main/java/softClass/homework/yurii_shevchuk/ShoppingCart.java:6]: Yes, I will push all previous homework to WednesdayHomeworks repo!

**Pull request closed without merging**